pkg/template/registry/templateinstance/strategy.go

```diff
@@ -55,9 +55,19 @@ func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj run
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
 	if templateInstance.Spec.Requester == nil {
-		user, _ := apirequest.UserFrom(ctx)
-		templateInstance.Spec.Requester = &templateapi.TemplateInstanceRequester{
-			Username: user.GetName(),
+		user, ok := apirequest.UserFrom(ctx)
+		if ok {
```

Comment:
It will essentially be like what the PR originally contained.  Per @jim-minter 's original question, we still check ok, but don't take any overt action if !ok.

Also, I think @liggitt 's comment https://github.com/openshift/origin/pull/15719#discussion_r132555854 lines up with my https://github.com/openshift/origin/pull/15719#discussion_r132555344

Again, we'll just remote the requester == nil conditional once the upstream SC update arrives and we can depend on user.Info.